react/Form.tsx

```diff
@@ -46,6 +46,27 @@ const Form: FC<FormProps> = props => {
 
   const schemaDocument = data.documentPublicSchema.schema
 
+  if (!schemaDocument) {
+    console.error('No MasterData Schema found')
+    return null
+  }
+
+  if (!('properties' in schemaDocument)) {
```

Comment:
Why not `!schemaDocument.properties`?

Reply:
Good question, I just learn that from @kaisermann and had to use it hahaha